Sources/UIKit+Hero.swift

```diff
@@ -117,6 +117,46 @@ internal extension NSObject{
 }
 
 public extension UIViewController{
+  private struct PreviousDelegates {
+    static var navigationDelegate: UINavigationControllerDelegate? = nil
```

Comment:
I merged too fast. Now I realized that this won't work. These two properties should be strings represent unique keys. Can you put another PR to fix this?